src/applications/vaos/new-appointment/components/ClinicChoicePage.jsx

```diff
@@ -78,10 +82,10 @@ export function ClinicChoicePage({
       scrollAndFocus();
       document.title = `${getPageTitle(schema, typeOfCare)} | Veterans Affairs`;
     },
-    [schema && facilityDetailsStatus !== FETCH_STATUS.loading],
+    [schemaAndFacilityReady],
```

Comment:
There was a linter warning about not using expressions in the dependency array, so I moved that to a variable.